app/javascript/packs/doc-capture-polling.ts

```diff
@@ -7,6 +7,9 @@ new DocumentCapturePolling({
   phoneQuestionAbTestBucket: document
     .querySelector('[data-phone-question-ab-test-bucket')
     ?.getAttribute('data-phone-question-ab-test-bucket') as string,
+  phoneWithCamera: document
+    .querySelector('[data-phone-with-camera')
```

Comment:
I didn't notice this until now, but the selector in the new (and existing!) `querySelector` here are invalid, so probably won't match anything.

Specifically there's a missing closing square bracket on both

```suggestion
    .querySelector('[data-phone-question-ab-test-bucket]')
    ?.getAttribute('data-phone-question-ab-test-bucket') as string,
  phoneWithCamera: document
    .querySelector('[data-phone-with-camera]')
```

Reply:
Ah. Thanks for catching that 😬 ([commit](https://github.com/18F/identity-idp/pull/9481/commits/ca4e6c0d30d1c6eab1c5847cbede474da1286b8c))